modules/rosa-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-upgrading.adoc
+
+[id="rosa-upgrading-cli_{context}"]
+= Upgrading with the rosa CLI
+
+[role="_abstract"]
+There are three methods that can be used to upgrade {product-title} clusters. This procedure describes upgrading using the `rosa` CLI.
+
+This method schedules the cluster for an immediate upgrade, if a more recent version is available.
+
+.Procedure
+. To verify that an upgrade is available, enter the following command:
++
+[source,terminal]
+----
+$ rosa list versions
+----
++
+A list displays the current version of your cluster, indicated by `true`, and a list of available versions. The following example shows that the current version is `v4.6.8`, and the most recent version is `v4.6.12`.
++
+.Example output
+[source,terminal]
+----
+ID                   DEFAULT
+openshift-v4.6.8     true
+openshift-v4.6.1     false
+openshift-v4.6.12    false
+openshift-v4.6.3     false
+openshift-v4.6.4     false
+openshift-v4.6.6     false
+----
+
+. To upgrade a cluster, enter the following command:
```

Comment:
Please start this section with a simpler example.  Just ```rosa upgrade cluster -c blah``` ... that will choose the latest available version.   You can follow that up with the --version example as written.

My reasoning is that we should be highlighting what we want the customer behavior to be, which is upgrading to the latest available version.

Reply:
Done. But I have removed the example with the `--version=4.6.12` because that doesn't make sense if we're telling them to use the simplest method, and because the simplest method is what we want the customer behavior to be. And the CLI reference guide already describes the `--version` flag.